src/libs/data-table-versions.js

```diff
@@ -13,32 +14,102 @@ import * as Utils from 'src/libs/utils'
 
 export const dataTableVersionsPathRoot = '.data-table-versions'
 
-export const saveDataTableVersion = async (workspace, entityType, { description = null } = {}) => {
+const getAllEntities = async (workspace, entityType) => {
+  const { workspace: { namespace, name } } = workspace
+
+  const pageSize = 100_000
```

Comment:
I don't recognize this syntax (100_000). Can you explain?

Reply:
Underscores can be used as separators in numeric literals. I think it's easier to read when there's a lot of zeros.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_separators